spec/unit/installer/xcode/pods_project_generator/file_references_installer_spec.rb

```diff
@@ -114,13 +114,29 @@ class PodsProjectGenerator
               )
             end
 
-            it "doesn't link public headers from vendored framework, when frameworks required" do
-              Target.any_instance.stubs(:requires_frameworks?).returns(true)
+            it 'does not link public headers from vendored framework, when frameworks required' do
+              @pod_target.stubs(:requires_frameworks?).returns(true)
               @installer.install!
               headers_root = config.sandbox.public_headers.root
               framework_header = headers_root + 'BananaLib/Bananalib/Bananalib.h'
               framework_header.should.not.exist
             end
+
+            it 'does not symlink headers that belong to test specs' do
+              coconut_spec = fixture_spec('coconut-lib/CoconutLib.podspec')
+              coconut_test_spec = coconut_spec.test_specs.first
+              coconut_pod_target = fixture_pod_target_with_specs([coconut_spec, coconut_test_spec], false)
+              public_headers_root = config.sandbox.public_headers.root
+              private_headers_root = @pod_target.build_headers.root
```

Comment:
whoops this should `coconut_pod_target` still worked though cause its the same path.